app/graphql/types/fees/amount_details/object.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module Types
+  module Fees
+    module AmountDetails
+      class Object < Types::BaseObject
+        graphql_name 'FeeAmountDetails'
+
+        # NOTE: Graduated charge model
+        field :graduated_ranges, [Types::Fees::AmountDetails::GraduatedRange], null: true
+
+        # NOTE: Graduated percentage modle
+        field :graduated_percentage_ranges, [Types::Fees::AmountDetails::GraduatedPercentageRange], null: true
+
+        # NOTE: Package charge model
+        field :free_units, String, null: true
+        field :paid_units, String, null: true
+        field :per_package_size, Integer, null: true
+        field :per_package_unit_amount, String, null: true
+
+        # NOTE: Percentage charge model
+        field :units, String, null: true
+        field :free_events, Integer, null: true
+        field :rate, String, null: true
+        field :per_unit_total_amount, String, null: true
+        field :paid_events, Integer, null: true
+        field :fixed_fee_unit_amount, String, null: true
+        field :fixed_fee_total_amount, String, null: true
```

Comment:
GraphQL/OrderedFields: Fields should be sorted in an alphabetical order within their section. Field `fixed_fee_total_amount` should appear before `fixed_fee_unit_amount`.